src/main/kotlin/org/ptss/support/api/controllers/ToolController.kt

```diff
@@ -14,12 +17,24 @@ import org.ptss.support.security.Authentication
 @Path("/tools")
 @ApplicationScoped
 @Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
 @Authentication(roles = [Role.PATIENT, Role.FAMILY_MEMBER, Role.HCP])
```

Comment:
Please do test this beforehand, but i realised that these could just as well be in the interface instead of here. Please do test this beforehand though :)

If this works, what kind of idiot made the interface in the base repo??? 
<sup>please dont check, please don't check, please don't check <sub>

Reply:
XD fair enough

Reply:
well does it work in the interface or nah?

Reply:
Ja doet het